src/MassTransit.RabbitMqTransport/Topology/RabbitMqExchangeBindingExtensions.cs

```diff
@@ -63,9 +67,9 @@ public static ExchangeBindingSettings GetExchangeBinding(this SendSettings setti
 
         public static IEnumerable<ExchangeBindingSettings> GetExchangeBindings(this ReceiveSettings settings, string exchangeName)
         {
-            var exchange = new Exchange(exchangeName, settings.Durable, settings.AutoDelete, settings.ExchangeType);
-
-            var binding = new ExchangeBinding(exchange);
+            var exchange = new Exchange(exchangeName, settings.Durable, settings.AutoDelete, settings.ExchangeTypeProvider);
+            //TODO find a way for this to stay working while you want to implement routingkey structure

```

Comment:
So this item is still pending to resolve. It's unclear to me what it's indicating though -- especially with the next line having a block comment in the middle.